src/test/java/redis/clients/jedis/tests/commands/AllKindOfValuesCommandsTest.java

```diff
@@ -488,6 +490,23 @@ public void dumpAndRestore() {
     assertTrue(jedis.exists("foo2"));
   }
 
+  @Test(expected = JedisBusyException.class)
```

Comment:
If we already have JedisBusyException I guess it's best to expect that one instead of a generic one